python-venv/cmstools/test/cfs_sessions_rc_test/argument_parser.py

```diff
@@ -0,0 +1,124 @@
+    """
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
+        "--run-subtests",
+        type=check_subtest_names,
+        help="Comma-separated list of subtests to run. Mutually exclusive with --skip-subtests. If neither is specified, all subtests are run."
+    )
+    group.add_argument(
+        "--skip-subtests",
+        type=check_subtest_names,
+        help="Comma-separated list of subtests to skip. Mutually exclusive with --run-subtests. If neither is specified, all subtests are run."
+    )
+
+def _add_cfs_session_arguments(parser: argparse.ArgumentParser) -> None:
+    """
+    Add arguments related to CFS sessions
+    --name	All sessions used for this test will have names with this prefix.
+    It should default to one that is not likely to be used by a real customer. Something like cfs-race-condition-test-.
+    """
+    parser.add_argument(
+        "--name",
+        type=validate_name,
+        default=DEFAULT_SESSION_NAME_PREFIX,
+        help="Prefix for all session names (default: %(default)s)"
+    )
+
+def _add_setup_arguments(parser: argparse.ArgumentParser) -> None:
+    """
+    Add arguments related to set up
+    --delete-previous-sessions	If true, the test will automatically delete any sessions that exist at the
+    start of the test that are in pending state and contain the specified name string. If false, if such sessions exist,
+    the test will exit in error.
+    --cfs-version	Which version of the CFS API to use. Defaults to 3.
+    --page-size	The page size to use for multi-get requests (default to 10*<max-sessions>).
+    If using CFS v2, the minimum value is <max-sessions>. Otherwise, minimum value of 1
+    If running CFS v2, set the V3 global CFS page-size option to <page-size>
+    (if it does not already have that value), but the original value should be restored when the test exits
+    """
+    parser.add_argument(
+        "--delete-previous-sessions",
+        action='store_true',
+        help="Delete any existing pending sessions with the specified name prefix"
+    )
+    parser.add_argument(
+        "--cfs-version",
+        type=str,
+        default=DEFAULT_CFS_VERSION,
+        choices=get_args(CFS_VERSIONS_STR),
+        help="CFS API version to use (default: %(default)s)"
+    )
+    parser.add_argument(
+        "--page-size",
+        type=check_min_page_size,
+        default=None,
+        help="Page size for multi-get requests (default: 10 * max-sessions for v3, min=max-sessions for v2, min=1 for v3)"
+    )
+
+def _add_subtests_arguments(parser: argparse.ArgumentParser) -> None:
+    """
+    Add arguments related to subtests
+     --max-sessions	Maximum number of CFS sessions to create (maybe start with a default of 20)
+    --max-multi-delete-reqs	Maximum number of parallel multi-delete requests (maybe start with a default of 4)
+    --max-multi-get-reqs	Maximum number of parallel multi-get requests (maybe start with a default of 4)
```

Comment:
The parenthetical remarks are not needed in the actual docstring -- they were just suggestions to you for the implementation.
```suggestion
     --max-sessions	Maximum number of CFS sessions to create
    --max-multi-delete-reqs	Maximum number of parallel multi-delete requests
    --max-multi-get-reqs	Maximum number of parallel multi-get requests
```